trader/trader.go

```diff
@@ -117,6 +122,11 @@ func (t *Trader) Start() {
 		currentUpdateTime := time.Now()
 		if lastUpdateTime.IsZero() || t.timeController.ShouldUpdate(lastUpdateTime, currentUpdateTime) {
 			success := t.update()
+			if success {
+				l := logger.MakeBasicLogger()
+				go t.tracker.SendUpdateEvent(l)
```

Comment:
We had discussed using the `threadTracker` workflow to kick off a goroutine here so it is "managed". Unless there's a good reason to use an ad-hoc goroutine in this case, I think we should use the `threadTracker` approach.

Reply:
We should also send this once every X seconds (which can be a hard-coded number set to 600 in `metricTrackingUpdateIntervalSeconds` for now). This calculation is done by diffing against the `lastUpdateTime`. With that in mind, the `SendUpdateEvent` function should take in the `currentUpdateTime` for use in it's event.

We will need a new module that returns `metricTrackingUpdateIntervalSeconds` that takes in the startTime like we had discussed (within the first 2 hours it returns a value of 15 minutes, and after that it returns a value of 1 hour)

Reply:
Ah, I definitely remember discussing the `threadTracker` usage - my bad. 

I don't think we discussed any new module or this differential behavior between the first 2 hours and later. Why do we want this approach?

Reply:
I didn't go into too much detail into the differential behavior because it's a nice to have and I didn't want us to lose focus from delivering the change. We can always add that as a separate commit/PR.
I'm fine with either including here or a separate PR, whatever you prefer.

And yes, a function works too.